app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -194,3 +194,9 @@ $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
 
 $select-element-padding:                        3px, 24px, 3px, 3px
+
+$homescreen-content-bg-color:                   white
+$homescreen-block-bg-color:                     white
```

Comment:
I'd also use a variable here. I was looking out for something like `$block-color`, but could not find anything... so probably also `$body-background`...